common/lib/capa/capa/templates/formulaequationinput.html

```diff
@@ -4,23 +4,14 @@
     <input type="text" name="input_${id}" id="input_${id}"
         data-input-id="${id}" value="${value|h}"
         aria-label="${label}"
+        aria-describedby="${id}_mathjax ${id}_status"
```

Comment:
@ahsan-ul-haq After testing, I no longer think it is a good idea to have the aria-describedby referencing the MathJax anymore.  Can you just leave the status in there?